guide/inheritance-wallet/onboarding-cosigners.md

```diff
@@ -0,0 +1,153 @@
+    - file: alice-enable-device/enable-alice-success
+      alt: Success screen showing the option activate another device or go to the home screen.
+      caption: On the success screen, the app shows the option to enable another signing device.
+    - file: alice-enable-device/enable-alice-completion
+      alt: Home screen showing the wallet with two tasks associated.
+      caption: On the home screen, Alice checks off the task. 
+    - file: alice-enable-device/enable-alice-completion-dialog
+      alt: Screen showing a confirmation dialog.
+      caption: The app asks whether she has enabled all signing devices.
+    - file: alice-enable-device/enable-alice-done
+      alt: Home screen showing the wallet with only one task associated.
+      caption: The last thing to do is to download the wallet backup kit.
+---
+
+<!--
+
+Editor's notes
+
+This page covers how Alice imports the savings wallet to her own app and enables her signing device to create and sign transactions.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1
+
+-->
+
+# Cosigner onboarding
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+In the previous step, Bob used our application to [create the wallet]({{ '/guide/inheritance-wallet/wallet-creation/' | relative_url }}). The next step is to get Alice set up as a cosigner and to enable her to sign transactions. To do this Alice needs to:
+
+1. Import the wallet configuration to her app
+2. Activate the wallet on her BitBox, by importing the wallet configuration to the device
+
+It's important to understand that importing the wallet to the phone does not enable Alice to sign transactions yet. Her signing key is held on her BitBox, as we have covered on the [wallet creation page]({{ '/guide/inheritance-wallet/wallet-creation/#step-2-adding-the-signing-keys' | relative_url }}). Both, the wallet app on her phone and the signing device need to know about the wallet configuration, so that they can work together to sign transactions. 
+
+#### Bob shares the wallet configuration with Alice
+
+The first step is to import the family savings wallet to Alice's app. By tapping the "onboard cosigner" task, Bob can display a QR code that Alice scans to import the wallet configuration. Alternatively, Bob can share the configuration over a secure messaging channel if Alice is elsewhere.
+
+{% include image-gallery.html pages = page.images_bob-export-config %}
+
+The app also shows an option to view the [wallet descriptor](https://bitcoin.design/guide/glossary/#output-script-descriptor), which is a small piece of text that contains all the information that a wallet application needs to recover a multi-key wallet. We will cover this topic in more detail in the following chapter about the wallet backup.
```

Comment:
Earlier on there is mention that the wallet config file is sometimes referred to as the descriptor. It might be easier to in this paragraph keep to one word as the title contains "configuration".